src/settings/project/mod.rs

```diff
@@ -83,6 +84,26 @@ fn get_project_config(config_path: &Path) -> Result<Project, failure::Error> {
     // Eg.. `CF_ACCOUNT_AUTH_KEY=farts` would set the `account_auth_key` key
     s.merge(Environment::with_prefix("CF"))?;
 
+    // check for pre 1.1.0 KV namespace format
+    let kv_namespaces: Result<Vec<config::Value>, config::ConfigError> = s.get("kv-namespaces");
+
+    if let Ok(values) = kv_namespaces {
+        let old_format = values.iter().any(|val| val.clone().into_str().is_ok());
+
+        if old_format {
+            message::warn("As of 1.1.0 the kv-namespaces format was stabilized");
+            message::info("Please add a section like this in your wrangler.toml for each KV Namespace you wish to bind:");
+
+            let fmt_demo = r#"
+[[kv-namespaces]]
+binding = "BINDING_NAME"
+id = "0f2ac74b498b48028cb68387c421e279"
+            "#;
+
+            println!("{}", fmt_demo);
+        }
```

Comment:
should this return early?

Reply:
nope, I want it to fall through to reuse the normal project error, which with the context about the changed binding format is informative

Reply:
reading the comment below, i don't think the normal project error necessarily gives any more context, in fact it's repetitive and therefore a little confusing to read.

Reply:
changed the message but I personally prefer the fall-through one as it explains the exact error in gory detail as well as how to fix it